integration-tests/tests/src/node/path.ts

```diff
@@ -36,24 +38,52 @@ const schema = {
   },
 };
 
+const FlexibleSchema = { ...Schema, properties: { ...Schema.properties, nonQueryable: "string?" } };
+
 describe("path configuration (local)", function () {
   it("relative path", function () {
     const filename = getRelativePath();
-    const realm = new Realm({ path: filename, schema: [schema] });
+    const realm = new Realm({ path: filename, schema: [Schema] });
     expect(realm.path.endsWith(filename)).to.be.true;
     realm.close();
     Realm.deleteFile({ path: filename });
   });
 
   it("absolute path", function () {
     const filename = getAbsolutePath();
-    const realm = new Realm({ path: filename, schema: [schema] });
+    const realm = new Realm({ path: filename, schema: [Schema] });
     expect(realm.path).to.equal(filename);
     realm.close();
     Realm.deleteFile({ path: filename });
   });
 });
 
+describe.skipIf(environment.missingServer, `app configuration of root directory (flexible sync)`, async function () {
+  const { appId, baseUrl } = await importApp("with-db-flx");
+
+  it.only("directory and file created where expected", async function () {
```

Comment:
Is the `only` intentional? (Thinking it may have been intended to be used temporarily?)
```suggestion
  it("directory and file created where expected", async function () {
```